src/js/stores/AvailableFiltersStore.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is part of thumbor-toy project.
+ *
+ * (c) Raphaël Benitte <thumbor-toy@rbenitte.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+import Reflux                 from 'reflux';
+import AvailableFilterActions from './../actions/AvailableFilterActions';
+import ConfigStore            from './../stores/ConfigStore';
+import baseFilters            from './../baseFilters';
+import _                      from 'lodash';
+
+var availableFilters = [];
+
+var AvailableFiltersStore = Reflux.createStore({
+    listenables: AvailableFilterActions,
+
+    init() {
+        this.listenTo(ConfigStore, this.update);
+        this.update();
+    },
+
+    update() {
+        availableFilters = [];
+        _.forEach(ConfigStore.get('filters'), function (filter) {
```

Comment:
You should use `Array.forEach()` instead as thumbor-toy doesn't support legacy browsers.